publications-2023.bib

```diff
@@ -508,6 +521,19 @@ @Article{2023:luo:fast
   publisher = {Elsevier {BV}}
 }
 
+@Article{2023:maier.shadid.tomas:euler-poisson,
+  author  = {Matthias Maier and John Shadid and Ignacio Tomas},
```

Comment:
Oh, you wrote a paper with John Shadid! I like John a lot!

Reply:
:smiley: